src/modules/product-category/product-category.controller.ts

```diff
@@ -0,0 +1,58 @@
+import { Controller, Get, Post, Body, Param, Patch, Delete, UseGuards, Query } from '@nestjs/common';
+import { ProductCategoryService } from './product-category.service';
+import { CreateProductCategoryDto } from './dto/create-product-category.dto';
+import { UpdateProductCategoryDto } from './dto/update-product-category.dto';
+import { ApiTags, ApiBody, ApiParam } from '@nestjs/swagger';
+import { skipAuth } from '../../helpers/skipAuth';
+@ApiTags('products/categories')
+@Controller('products/categories')
+export class ProductCategoryController {
+  constructor(private readonly categoryService: ProductCategoryService) {}
+
+  @Get()
+  async findAll(@Query('limit') limit?: number, @Query('offset') offset?: number) {
```

Comment:
instead of having multiple Query() parameters here you can get the params, you can pass just the Req() parameter and get your query parameters from request.query

Reply:
okay.